mne/evoked.py

```diff
@@ -561,14 +562,22 @@ def get_peak(self, ch_type=None, tmin=None, tmax=None,
             Whether to return the time index instead of the latency in seconds.
         merge_grads : bool
             If True, compute peak from merged gradiometer data.
+        return_amplitude : bool
+            If True, return also the amplitude at the maximum response.
 
+        .. versionadded:: 0.16
```

Comment:
needs to be indented (same level as the rest of the `return_amplitude` description), and needs a newline after it (it broke the `Returns` section)

https://ci.appveyor.com/project/mne-tools/mne-python/build/1.0.1146#L472